test/distributed/pipelining/artifacts/zb1p_2rank_2stagep_compute.csv

```diff
@@ -0,0 +1,2 @@
+0F0,0F1,2F0,,2F1,2I0,2W0,0F2,2I1,2W1,0F3,0I0,0W0,2F2,0I1,0W1,2F3,2I2,2W2,0F4,2I3,2W3,0F5,0I2,0W2,2F4,0I3,0W3,2F5,2I4,2W4,0F6,2I5,2W5,0F7,0I4,0W4,2F6,0I5,0W5,2F7,2I6,2W6,2I7,2W7,0I6,0W6,0I7,0W7
```

Comment:
For my understanding, if you load the compute csv without the comms csv will it error? Or will it automatically determine the comms for you

Reply:
within test_csv, it is explicit: we load the compute-only one, run the lowering passes, then compare the output of that with the saved comms one.

for real users, it is API'd somewhat: load_csv in PipelineScheduleRuntime accepts a kwarg for whether its a comms or compute csv.  If its a compute one, then it will run add_send_recv.

I think before we more widely roll this out, we should better define the api around the lowering passes.  Probably a function for 'lowering' the schedule and some config flags for any optional passes.  For now its kind of manual: load_csv just hardcodes which passes to run.